packages/flutter/lib/src/material/chip.dart

```diff
@@ -1368,61 +1368,66 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     final TextDirection textDirection = Directionality.of(context);
     final ShapeBorder shape = widget.shape ?? chipTheme.shape;
 
-    return new Material(
-      elevation: isTapping ? _kPressElevation : 0.0,
-      animationDuration: pressedAnimationDuration,
-      shape: shape,
-      child: new InkResponse(
-        onTap: canTap ? _handleTap : null,
-        onTapDown: canTap ? _handleTapDown : null,
-        onTapCancel: canTap ? _handleTapCancel : null,
-        child: new AnimatedBuilder(
-          animation: new Listenable.merge(<Listenable>[selectController, enableController]),
-          builder: (BuildContext context, Widget child) {
-            return new Container(
-              decoration: new ShapeDecoration(
-                shape: shape,
-                color: getBackgroundColor(chipTheme),
-              ),
-              child: child,
-            );
-          },
-          child: _wrapWithTooltip(
-            widget.tooltip,
-            widget.onPressed,
-            new _ChipRenderWidget(
-              theme: new _ChipRenderTheme(
-                label: new DefaultTextStyle(
-                  overflow: TextOverflow.fade,
-                  textAlign: TextAlign.start,
-                  maxLines: 1,
-                  softWrap: false,
-                  style: widget.labelStyle ?? chipTheme.labelStyle,
-                  child: widget.label,
-                ),
-                avatar: new AnimatedSwitcher(
-                  child: widget.avatar,
-                  duration: _kDrawerDuration,
-                  switchInCurve: Curves.fastOutSlowIn,
+    return new Semantics(
+      container: true,
+      selected: widget.selected,
+      enabled: widget.onPressed != null ? widget.isEnabled : null,
```

Comment:
You also need to check if "onSelected" is non-null, since either onPressed or onSelected can be set.  What I think you really want is to check the "canTap" accessor.